src/aks-preview/azext_aks_preview/__init__.py

```diff
@@ -16,7 +16,7 @@ class ContainerServiceCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
-        register_resource_type('latest', CUSTOM_MGMT_AKS_PREVIEW, '2020-04-01')
+        register_resource_type('latest', CUSTOM_MGMT_AKS_PREVIEW, '2020-06-01')
```

Comment:
pls make sure you do full LIVE test regression when bumping API version

Reply:
could you elaborate a little? not sure what i need to do as LIVE test regression

Reply:
this change bumping major API version. there's could be breaking change cross different API versions. Run existing tests against real AKS could help to find out these change to make sure no regression in existing CLI commands.

Reply:
got it. will run the existing test cli commands and check if everything is creating properly